### PR TITLE
Reworked job_control.py

### PR DESCRIPTION
**Pull request by intendo**
@mmacduff can you test the new job_control.py to see if it still works in the image

### CODE REVIEW

File: src/docker/job_control.py [hunk 1]
```diff
@@ -1,132 +1,245 @@
...
+AWS_REGION = "us-west-2"
+MAX_TRIES = 60
+RETRY_TIME = 15
+
+MEM_PATH = "/dev/shm/test/"
+S3_BUCKET = getenv("OUTPUT_BUCKET")
+QUEUE = getenv("JOB_QUEUE_NAME")
+
+
+class JOBTYPE(Enum):
```
Comment (mmacduff) [inline src/docker/job_control.py:26]: the rest of the code/web site uses lower case for these types.  
And i believe that's for the JOBSTATUS too.   The web is exepcted lower case strings.

Comment (mmacduff) [reply src/docker/job_control.py:26]: at a minimum I just did lc on these 2 to get the backend to work
And also lc on the runjob check for these names.

File: src/docker/job_control.py [hunk 2]
```diff
@@ -1,132 +1,245 @@
...
+            QueueUrl=qurl, MaxNumberOfMessages=1, VisibilityTimeout=Q_TIMEOUT
+        )
+
+    return items
+
+
+def update_status(
+    s3client: client,
+    jobid: str,
+    jobtype: JOBTYPE,
```
Comment (mmacduff) [inline src/docker/job_control.py:71]: this is actually a type str being passed in.
which means that line 86 should not have .name

**Review by mmacduff — CHANGES_REQUESTED** [review]
Added a couple comments that need to be fixed for it to execute.

File: src/docker/job_control.py (hunk 1 above)
Comment (intendo) [reply src/docker/job_control.py:31]: I think we fixed this.

File: src/docker/job_control.py (hunk 2 above)
Comment (intendo) [reply src/docker/job_control.py]: I think this is fixed too.

**Review by mmacduff — APPROVED** [review]

**Comment by intendo** [comment]
Closes #26

**Pull request merged**